docs/tools/developer-tools/lab/api-explorer/horizon-endpoint.mdx

```diff
@@ -0,0 +1,360 @@
+        "asset_type": "native",
+        "from": "GBPIMUEJFYS7RT23QO2ACH2JMKGXLXZI4E5ACBSQMF32RKZ5H3SVNL5F",
+        "to": "GDE25LQ34AFCSDMYTOI6AVVEHRXFRJI4MOAVIUGUDUQEC5ZWN5OZDLAZ",
+        "amount": "4.0000000"
+      },
+      {
+        "_links": {
+          "self": {
+            "href": "https://horizon-testnet.stellar.org/operations/4115390418391041"
+          },
+          "transaction": {
+            "href": "https://horizon-testnet.stellar.org/transactions/6de546bd7702f1590f21928e6aa1329582b436f11d9fc9bcce304ea51f1893b0"
+          },
+          "effects": {
+            "href": "https://horizon-testnet.stellar.org/operations/4115390418391041/effects"
+          },
+          "succeeds": {
+            "href": "https://horizon-testnet.stellar.org/effects?order=desc&cursor=4115390418391041"
+          },
+          "precedes": {
+            "href": "https://horizon-testnet.stellar.org/effects?order=asc&cursor=4115390418391041"
+          }
+        },
+        "id": "4115390418391041",
+        "paging_token": "4115390418391041",
+        "transaction_successful": true,
+        "source_account": "GBPIMUEJFYS7RT23QO2ACH2JMKGXLXZI4E5ACBSQMF32RKZ5H3SVNL5F",
+        "type": "payment",
+        "type_i": 1,
+        "created_at": "2024-11-14T21:12:31Z",
+        "transaction_hash": "6de546bd7702f1590f21928e6aa1329582b436f11d9fc9bcce304ea51f1893b0",
+        "asset_type": "native",
+        "from": "GBPIMUEJFYS7RT23QO2ACH2JMKGXLXZI4E5ACBSQMF32RKZ5H3SVNL5F",
+        "to": "GCNJ2TA24PJNDPE2JF3BYVLUTJX2GLRDGGBG4SYMQHKHYDHSMAPRZXWC",
+        "amount": "5.0000000"
+      }
+    ]
+  }
+}
+```
+
+### [All Assets](https://lab.stellar.org/endpoints/assets)
+
+Horizon's `/assets` endpoint provides information about an asset. Since we are not interested in historical data, we will use SDF's mainnet Horizon to retrieve the latest information about the asset.
+
+![Lab: Horizon - Assets Page](/assets/lab/horizon-assets.png)
+
+Let's inquire about one of the most popular assets on the Stellar network: [USDC by Circle](https://www.circle.com/usdc). To do this, enter `USDC` as the Asset Code and `GA5ZSEJYB37JRC5AVCIA5MOP4RHTM335X2KGX3IHOJAPP5RE34K4KZVN` as the Asset Issuer.
+
+![Lab: Horizon - Assets Page](/assets/lab/horizon-assets-usdc.png)
+
+When I click the **Submit** button, the `/assets` endpoint returns the `USDC` asset details in `JSON` format.
+
+![Lab: Horizon - Assets Page](/assets/lab/horizon-assets-usdc-response.png)
+
+The JSON response for the `USDC` asset includes the following. Its records return [the asset object](https://developers.stellar.org/docs/data/horizon/api-reference/resources/assets/object), which contains details about USDC, such as the asset's TOML file (also known as [the Stellar info file](https://developers.stellar.org/docs/tokens/publishing-asset-info)), the number of claimable balances, liquidity pools, contracts, accounts, balances, and flags.
```

Comment:
The beginning sounds unclear to me. Maybe we could combine the first two sentences?